packages/nc-gui/components/smartsheet/column/FormulaOptions.vue

```diff
@@ -249,18 +249,18 @@ function validateAgainstMeta(parsedTree: any, errors = new Set(), typeErrors = n
             formulaTypes.DATE,
             (v: any) => {
               if (!validateDateWithUnknownFormat(v)) {
-                typeErrors.add('The second parameter of DATEADD() should have date value')
+                typeErrors.add('The second parameter of DATETIME_DIFF() should have date value')
               }
             },
             typeErrors,
           )
-           // parsedTree.arguments[2] = ["millisecond" | "ms" | "second" | "s" | "minute" | "mi" | "hour" | "hh" | "day" | "dy" | "week" | "wk" | "month" | "m" | "quarter" | "q" | "year" | "y"]
+           // parsedTree.arguments[2] = ["millisecond" | "ms" | "second" | "s" | "minute" | "mi" | "hour" | "hh" | "day" | "dy" | "week" | "wk" | "month" | "m" | "quarter" | "q" | "year" | "yy"]
```

Comment:
Please follow the below list.

["milliseconds", "seconds", "minutes", "hours", "days", "weeks", "months", "quarters", "years", "ms", "s", "m", "h", "d", "w", "M", "Q", "y"]